### DISCUSSION TITLE
Is there a snap to grid feature?

### DISCUSSION THREAD

**Discussion by nooblyf** [fabricjs/fabric.js#8827] [Q&A] [answered]
Is there an API to enable snap to grid for canvas objects? something like [this](https://konvajs.org/docs/sandbox/Objects_Snapping.html)

**Comment by ShaMan123** [comment, marked as answer]
checkout [aligning/centering guidelines](https://github.com/fabricjs/fabric.js/tree/d134c4f71e11ec2763eafa773793267ebd48a1c4/lib)
